core-bundle/src/Resources/contao/languages/en/tl_content.xlf

```diff
@@ -641,6 +641,9 @@
       <trans-unit id="tl_content.vimeo_byline">
         <source>Hide the author</source>
       </trans-unit>
+      <trans-unit id="tl_content.vimeo_dnt">
+        <source>Set to "Do Not Track"</source>
```

Comment:
How about something like that?
```suggestion
        <source>Prevent session data tracking (dnt)</source>
```

From the Vimeo docs <https://developer.vimeo.com/player/sdk/embed#table-1>
> Whether to prevent the player from tracking session data, including cookies. Keep in mind that setting this argument to true also blocks video stats.

Reply:
Yes, this is more meaningful.